src/providers/postgres/raster/qgspostgresrasterprovider.cpp

```diff
@@ -1185,6 +1228,46 @@ bool QgsPostgresRasterProvider::init()
     return false;
   }
 
+  // Temporal capabilities
+  // Setup temporal properties for layer
+  if ( mUri.hasParam( QStringLiteral( "temporalField" ) ) )
+  {
+    Q_ASSERT_X( temporalCapabilities(), "QgsPostgresRasterProvider::QgsPostgresRasterProvider", "Data provider temporal capabilities object does not exist" );
```

Comment:
I don't think this is explicitly required -- the temporalCapabilities object is always created by QgsRasterDataProvider